demos/textfield.html

```diff
@@ -129,7 +136,67 @@
         </p>
       </section>
       <section class="example">
-        <h2>CSS Only</h2>
+        <h2>Textfield box</h2>
+        <div id="demo-tf-box-wrapper">
+          <div id="tf-box-example" class="mdc-textfield mdc-textfield--box" data-demo-no-auto-js>
+            <input type="text" id="tf-box" class="mdc-textfield__input" required
+                   aria-controls="name-validation-message">
+            <label for="tf-box" class="mdc-textfield__label">Your Name</label>
+            <div class="mdc-textfield__bottom-line"></div>
+          </div>
+          <p class="mdc-textfield-helptext mdc-textfield-helptext--validation-msg"
+             id="name-validation-msg">
+            Must provide a name
+          </p>
+        </div>
+        <div>
+          <input id="box-disable" type="checkbox">
+          <label for="box-disable">Disabled</label>
+        </div>
+        <div>
+          <input id="box-rtl" type="checkbox">
+          <label for="box-rtl">RTL</label>
+        </div>
+        <div>
+          <input id="box-dark-theme" type="checkbox">
+          <label for="box-dark-theme">Dark Theme</label>
+        </div>
+        <div>
+          <input id="box-dense" type="checkbox">
+          <label for="box-dense">Dense</label>
+        </div>
+        <script>
+          setTimeout(function() {
+            var tfEl = document.getElementById('tf-box-example');
+            var tf = new mdc.textfield.MDCTextfield(tfEl);
+            var wrapper = document.getElementById('demo-tf-box-wrapper');
+
+            document.getElementById('box-disable').addEventListener('change', function(evt) {
+              tf.disabled = evt.target.checked;
+            });
+
+            document.getElementById('box-rtl').addEventListener('change', function(evt) {
+              if (evt.target.checked) {
+                wrapper.setAttribute('dir', 'rtl');
+              } else {
+                wrapper.removeAttribute('dir');
+              }
+              tf.ripple.layout();
+            });
+
+            document.getElementById('box-dark-theme').addEventListener('change', function(evt) {
+              wrapper.classList[evt.target.checked ? 'add' : 'remove']('mdc-theme--dark');
+            });
+
+            document.getElementById('box-dense').addEventListener('change', function(evt) {
+              tfEl.classList[evt.target.checked ? 'add' : 'remove']('mdc-textfield--dense');
+              tf.ripple.layout();
+            });
+          }, 80);
```

Comment:
Yeah agreed that 0 is better; I didn't realize we could use 0. Hopefully sometime soon we'll have a more robust solution for loading these demo styles.